planner/core/rule_join_reorder_greedy.go

```diff
@@ -93,6 +95,21 @@ func (s *joinReorderGreedySingleGroupSolver) solve() (LogicalPlan, error) {
 	return s.makeBushyJoin(cartesianGroup), nil
 }
 
+func (s *joinReorderGreedySingleGroupSolver) nodeCost(p LogicalPlan) float64 {
+	if cst, ok := s.groupNodeCst[p.ID()]; ok {
```

Comment:
how about changing `cst`/`Cst` to `cost`/`Cost` to unify the variable naming style?

Reply:
😂 Actually, there're many places using `cst` in planner package. If we need this change, I can do it in a separate pr, changing all `cst` and `Cst`.

Reply:
OK